src/libs/ReportUtils.js

```diff
@@ -993,6 +993,17 @@ function getTransactionReportName(reportAction) {
     });
 }
 
+/*
+ * There are report comments that are added by our system. Those messages are created in english.
+ * This function translates these messages before showing them in the LHN or in another option list.
+ */
+function getTranslatedReportLastMessage(report) {
+    if (isReportMessageAttachment({text: report.lastMessageText, html: report.lastMessageHtml})) {
+        return `[${Localize.translateLocal('common.attachment')}]`;
+    }
+    return report.lastMessageText || '';
```

Comment:
@luacmartins is this ok? or still prefer lodashget?

Reply:
that's fine